lib/refocus.rb

```diff
@@ -3,6 +3,7 @@
 
 module Refocus
   def self.client(url: ENV.fetch("REFOCUS_URL"), token: ENV.fetch("REFOCUS_API_TOKEN"))
-    Client.new(url: url, token: token)
+    @@client ||= Client.new(url: url, token: token)
+    @@client
```

Comment:
I don't think we should be doing this sort of caching here - this can be done by the client code.

The problem is that this method accepts parameters. If the user passes different parameters later, they'll still get back the same client object using the original parameters that they passed.

I'd prefer to just leave caching of the client, if desired, to the caller.

Reply:
ah, you're right, i was just letting the default args to be set so didn't think about that use case